src/app/api/Api.ts

```diff
@@ -11,6 +11,7 @@ export type AssignmentMap = { [key: number]: Assignment };
 export type AssignmentDutyMap = { [key: number]: AssignmentDuty };
 export type WorkSectionId = 'COURTS' | 'JAIL' | 'ESCORTS' | 'OTHER';
 export type Assignment = CourtAssignment | JailAssignment | EscortAssignment | OtherAssignment;
+export type ShiftCopySelection = 'shiftsOnly' | 'shiftsAndSheriffs';
```

Comment:
Do you ever use both of these options?  It seems like it could be replaced by a boolean `copySheriffs?` since either way your copying shifts.

Reply:
When I first started working on it, I was trying to use a boolean, but in the onSubmit, you get back the value of the checkbox (i.e. 'shiftsOnly' and 'shiftsAndSheriffs') so instead of doing a bunch of conversion, I went with a type.

Reply:
Hmm I see, I think a boolean would be most desirable as it avoids us having to express this type in the backend as well.

Reply:
I agree, perhaps lets take a look at this together